google/cloud/bigquery/external_config.py

```diff
@@ -760,6 +760,14 @@ def schema(self):
         prop = self._properties.get("schema", {})
         return [SchemaField.from_api_repr(field) for field in prop.get("fields", [])]
 
+    @property
+    def connection_id(self):
```

Comment:
Will likely need the standard "[Experimental]" disclaimer, as this feature is only available to certain customers in a private Preview at the moment.